plugin/cmd/kube-scheduler/app/configurator.go

```diff
@@ -143,32 +150,41 @@ func (sc schedulerConfigurator) getSchedulerPolicyConfig() (*schedulerapi.Policy
 	if !sc.useLegacyPolicyConfig && len(sc.policyConfigMap) != 0 {
 		namespace := sc.policyConfigMapNamespace
```

Comment:
make sure a namespace was specified... list and watch do unfortunate things when given an empty namespace